authenticators/cf_authenticator_test.go

```diff
@@ -155,6 +199,17 @@ var _ = Describe("CFAuthenticator", func() {
 			Expect(metadata).To(Equal(metadata))
 		})
 
+		It("logs the access to the container by the user", func() {
+			Eventually(logger).Should(gbytes.Say("test.cf-authenticate.app-access-success"))
+			for _, msg := range logger.Logs() {
+				if msg.Message == "test.cf-authenticate.app-access-success" {
+					Expect(msg.Data["app"]).To(Equal("1e051b88-a210-40b7-bcca-df645b24b634/1"))
+					Expect(msg.Data["username"]).To(Equal("admin"))
+					Expect(msg.Data["principal"]).To(Equal("36ba11ff-0f6a-4c50-ab34-6fbd286a643e"))
+				}
+			}
```

Comment:
One of the advantages of using `gbytes.Say` now is that you don't need to do this for loop anymore. You can reduce the whole test to a single assertion that looks something like this:

```
Eventually(logger).Should(
gbytes.Say("test.cf-authenticate.app-access-success: {\"app\": \"that-app-guid\", \"username\": \"admin\", ...}"))```